test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -792,6 +792,22 @@ public void editFile(String filename) {
         }
     }
     
+    public void editFile(String filename, String comment) {
```

Comment:
Why is this file being changed in this PR? The change in the file doesn't appear to have anything to do with the purpose of the PR...